hs_core/templates/pages/my-resources.html

```diff
@@ -96,23 +96,12 @@ <h4 class="panel-title">
                                     <div class="dropdown-menu dropdown-menu-left shadow-md" role="menu">
                                         <form class="form-horizontal" role="form">
                                             <div class="form-group">
-                                                <label for="filter">Resource Type</label>
+                                                <label for="filter">Type</label>
                                                 <select id="input-resource-type" class="form-control pull-left">
                                                     <option value="" selected>All</option>
                                                     <option value="collection resource">Collection</option>
-                                                    <option value="composite resource">Composite Resource</option>
-                                                    <option value="generic">Generic</option>
```

Comment:
"Generic resource" here need to change?
https://github.com/hydroshare/hydroshare/blob/4853-resource-type-name-fix/hs_core/templates/pages/share-resource.html#L12

Reply:
Yeah need to clean all the old resource types here.

Reply:
Good question. Not sure where/how this template is being used.

Reply:
Good call, I don't see that it is used...